apps/rpc/src/modules/user/user-router.ts

```diff
@@ -187,23 +199,21 @@ const updateUserProcedure = authenticatedProcedure
 export type IsStaffInput = inferProcedureInput<typeof isStaffProcedure>
 export type IsStaffOutput = inferProcedureOutput<typeof isStaffProcedure>
 const isStaffProcedure = procedure.use(withDatabaseTransaction()).query(async ({ ctx }) => {
```

Comment:
this is not part of the pr but why does this force using transactions?

Reply:
will be done in another PR